app/assess/routes.py

```diff
@@ -27,14 +28,20 @@
     "/application_id/<application_id>/sub_criteria_id/<sub_criteria_id>",
     methods=["POST", "GET"],
 )
-@login_required(roles_required=["COMMENTER"])
 def display_sub_criteria(
     application_id,
     sub_criteria_id,
 ):
     """
     Page showing sub criteria and themes for an application
     """
+
+    displayCommentBox = False
+    if 'add-comment' in request.args.keys() and request.args['add-comment'] == '1':
+        displayCommentBox = True            
+
+    commentForm = CommentsForm()
```

Comment:
Can we use `camel_case` to be consistent with the rest of the code base?

Reply:
Done and changed commentForm to comment_form as well as a few others